charts/auth/values.yaml

```diff
@@ -0,0 +1,103 @@
+# Default values for auth.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/formancehq/auth
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+auth:
+  debug: 1
+  postgresURI: "host=postgres user=formance password=formance dbname=auth port=5432 sslmode=disable"
+  caosOIDCdev: 1
+  baseUrl: http://localhost/api/auth
+  delegated:
+    clientID: ""
+    clientSecret: ""
+    issuer: ""
+  congigurationFilePath: /config.yaml
+  config:
+    - public: false
```

Comment:
This config will add a default client.
Actually, the server does not reject the entry but it will soon.
I think you should comment this section add set an empty array.